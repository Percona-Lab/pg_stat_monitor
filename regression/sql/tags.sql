CREATE EXTENSION pg_stat_monitor;
SELECT pg_stat_monitor_reset();
SELECT 1 AS num /* { "application", psql_app, "real_ip", 192.168.1.3) */;
SELECT query, comments FROM pg_stat_monitor ORDER BY query COLLATE "C";
ALTER SYSTEM SET pg_stat_monitor.extract_comments TO 'yes';
SELECT pg_reload_conf();
select pg_sleep(1);
SELECT pg_stat_monitor_reset();
SELECT 1 AS num /* { "application", psql_app, "real_ip", 192.168.1.3) */;
SELECT query, comments FROM pg_stat_monitor ORDER BY query COLLATE "C";
ALTER SYSTEM SET pg_stat_monitor.extract_comments TO 'no';
SELECT pg_reload_conf();
SELECT pg_stat_monitor_reset();
DROP EXTENSION pg_stat_monitor;
