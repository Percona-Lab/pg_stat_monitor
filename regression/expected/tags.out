CREATE EXTENSION pg_stat_monitor;
SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

SELECT 1 AS num /* { "application", psql_app, "real_ip", 192.168.1.3) */;
 num 
-----
   1
(1 row)

SELECT query, comments FROM pg_stat_monitor ORDER BY query COLLATE "C";
                                   query                                   | comments 
---------------------------------------------------------------------------+----------
 SELECT $1 AS num /* { "application", psql_app, "real_ip", 192.168.1.3) */ | 
 SELECT pg_stat_monitor_reset()                                            | 
 SELECT query, comments FROM pg_stat_monitor ORDER BY query COLLATE "C"    | 
(3 rows)

ALTER SYSTEM SET pg_stat_monitor.extract_comments TO 'yes';
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

select pg_sleep(1);
 pg_sleep 
----------
 
(1 row)

SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

SELECT 1 AS num /* { "application", psql_app, "real_ip", 192.168.1.3) */;
 num 
-----
   1
(1 row)

SELECT query, comments FROM pg_stat_monitor ORDER BY query COLLATE "C";
                                   query                                   |                         comments                         
---------------------------------------------------------------------------+----------------------------------------------------------
 SELECT $1 AS num /* { "application", psql_app, "real_ip", 192.168.1.3) */ | /* { "application", psql_app, "real_ip", 192.168.1.3) */
 SELECT pg_stat_monitor_reset()                                            | 
 SELECT query, comments FROM pg_stat_monitor ORDER BY query COLLATE "C"    | 
(3 rows)

ALTER SYSTEM SET pg_stat_monitor.extract_comments TO 'no';
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

DROP EXTENSION pg_stat_monitor;
